src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -78,6 +86,24 @@ public void run() {
 		}
 	}
 
+	private void inputDataInformation( final ReviewResult reviewResult ) throws IOException {
+		final InputStreamReader inputStreamReader = new InputStreamReader( System.in );
+		final BufferedReader bufferedReader = new BufferedReader( inputStreamReader );
+		final String input = bufferedReader.readLine();
+		if ( input.toLowerCase().equals( "yes" ) ) {
+			for ( final SuiteChangeSet suiteChangeSet : reviewResult.getSuiteChangeSets() ) {
+				applyChanges( createSutStatePersistence(), suiteChangeSet );
+			}
+			bufferedReader.close();
+		} else if ( input.toLowerCase().equals( "no" ) ) {
+			logger.info( "No changes are applied!" );
+			bufferedReader.close();
+		} else {
+			logger.info( "Invalid input! Please try one more time:" );
```

Comment:
Don't we want to have this in an infinite loop until the user has entered valid input?

Reply:
It works exactly so, but without the inifite loop. The log message also occurs every time, until the user has entered valid input.